scripts/whisper/find_alignment_heads.py

```
@@ -0,0 +1,281 @@
+    audio_features = model.encoder(mel.unsqueeze(0))
+
+    # Get tokenizer
+    tokenizer = whisper.tokenizer.get_tokenizer(
+        model.is_multilingual,
+        num_languages=getattr(model, 'num_languages', None) or (99 if model.is_multilingual else None),
+        task="transcribe",
+    )
+
+    # Initial tokens (SOT sequence + no_timestamps)
+    # The no_timestamps token is required for proper decoding
+    tokens = list(tokenizer.sot_sequence) + [tokenizer.no_timestamps]
```

Ruff (0.14.13)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)